cloud_watch_client.py

```diff
@@ -153,13 +153,19 @@ def get_alarms(self, state="ALARM"):
         res = []
         my_conf = dict(
             StateValue=state,
+            MaxRecords=100
         )
         if self._namespace:
             # Cloud watch does not accept an empty sting for AlarmNamePrefix
             my_conf['AlarmNamePrefix'] = self._get_full_normalized_alarm_name('')
         try:
-            resp = self._client.describe_alarms(**my_conf)
-            res = resp.get('MetricAlarms')
+            resp = None
+            next_token = None
+            while resp is None or next_token is not None:
+                resp = self._client.describe_alarms(**my_conf)
+                res.extend(resp.get('MetricAlarms', []))
+                next_token = resp.get('NextToken')
```

Comment:
if it might not return None, we can convert it like 
`next_token = resp.get('NextToken', None)`